docs/user-guide/media-editor-window.md

```diff
@@ -45,16 +45,44 @@ With a click on **Save** <img src="../img/button/save-icon.jpg" class="ms-docbut
 
 Once selectd in the list, the image can be included in the story by clicking on the **Apply** button <img src="../img/button/apply-button2.jpg" class="ms-docbutton"/>.
 
+## Videos
+
+In order to add a video, the stroy editor can click on the **Video** tab <img src="../img/button/videos.jpg" class="ms-docbutton"/> in order to switch to the videos section. In this section of the media editor window, with a click on the **Add** button <img src="../img/button/+++.jpg" class="ms-docbutton"/>, it is possible to define the video settings.
+
+<img src="../img/media-editor-window/video-settings.jpg" class="ms-docimage"/>
+
+In particular, it is possible to insert the following parameters:
+
+* The **Video URL** of the video
+
+* The **Title** of the video
+
+* A **Description**, used to explain the contents of the video. The description is available in the viedos preview list and, if the video is added as content in the *Pharagraph section*, in the *Immersive section* or in the *Media section*, under the video
+
+* The **Credits**, displayed on the bottom-right of the image
+
+!!! warning
+    **Video URL** and **Title** are mandatory fields.
+
+With a click on **Save** <img src="../img/button/save-icon.jpg" class="ms-docbutton"/>, the video is included in the list of the available videos ready to be selected for the current story. The video also becomes immediately available on preview.
+
+<img src="../img/media-editor-window/video-preview.jpg" class="ms-docimage"/>
+
+Once selectd in the list, the video can be included in the story by clicking on the **Apply** button <img src="../img/button/apply-button2.jpg" class="ms-docbutton"/>.
+
 ## Maps
 
 To add a map to the story, the story editor can click on the **Maps** tab <img src="../img/button/maps-button.jpg" class="ms-docbutton"/> and so swich to the map section of the Media Editor. Here the list of maps ready to be used for the story is available: the editor can search and select a map in the list to apply it in the story or create a map from scratch by clicking on the **Add** button <img src="../img/button/+++.jpg" class="ms-docbutton"/> to open the [Map Editor](configure-map.md#advanced-map-editor).
 
 <img src="../img/media-editor-window/edit-map.jpg" class="ms-docimage"/>
 
-Once the map is ready, the editor can click on Ok to proceed with the next step and therefore insert the related map metadata like *Thumbnail*, the *Title* and the *Description*.
+Once the map is ready, the editor can click on Ok to proceed with the next step and therefore insert the related map metadata like **Thumbnail**, the **Title** and the **Description**.
 
 <img src="../img/media-editor-window/save-map-backg.jpg" class="ms-docimage"/>
 
+!!! note
+    The description is available in the maps preview list and, if the map is added as content in the *Pharagraph section*, in the *Immersive section* or in the *Media section*, under the map.
```

Comment:
The description is available in the maps preview list and it can appear under the map itself if the map is added as a content of a section (*Pharagraph section*, *Immersive section* or *Media section*).